tasks/ethernet_configuration.yml

```diff
@@ -1,6 +1,6 @@
 ---
-
 - name: Check active Ethernet interface state
+  become: true
```

Comment:
I don't think it's necessary for this task.

Reply:
I fixed this too. thanks <3